build/config/rmi_pacta_2023q4_pa2024ch.json

```
@@ -3,7 +3,7 @@
   "index_share_path": "2023Q4_20240301T145404Z",
   "pacta_data_quarter": "2023Q4",
   "project_code": "PA2024CH",
-  "templates_ref": "PA2024CH-templates",
+  "templates_ref": "add-templates-cop2024ch",
```

since https://github.com/RMI-PACTA/templates.transition.monitor/pull/11 is merged, I guess this can now be
```suggestion
  "templates_ref": "",
```